static_src/actions/login_actions.js

```diff
@@ -5,13 +5,40 @@
  */
 
 import AppDispatcher from '../dispatcher.js';
+import cfApi from '../util/cf_api';
 import { loginActionTypes } from '../constants';
 
-export default {
-  receivedStatus(status) {
+const loginActions = {
+  fetchStatus() {
+    AppDispatcher.handleViewAction({
+      type: loginActionTypes.FETCH_STATUS,
+      status
```

Comment:
Ah, good catch. Since it's a fetch, we don't have any auth status, so this is undefined no matter what and should just be removed.